tests/test_helpers.py

```diff
@@ -30,14 +30,16 @@ def test_generate_index_name():
 
 def test_generate_step_output_filename_with_sequence():
     assert (
-        helpers.generate_step_output_filename("index", "prefix", "extract", "01")
+        helpers.generate_step_output_filename(
+            "source", "index", "prefix", "extract", "01"
+        )
         == "prefix-to-index_01.xml"
     )
```

Comment:
This test is helpful that is confirms a `.xml` filename is created by default.  Would you be willing to add a test that confirms both `gismit` and `gisogm` sources get a `.jsonl` extension?

Reply:
See latest commit!